### PR TITLE
Mobile 4498

### PR DESCRIPTION
**Pull request by crazyserver**

### CODE REVIEW

File: src/core/features/courses/services/courses-helper.ts [hunk 1]
```diff
@@ -437,6 +439,18 @@ export class CoreCoursesHelperProvider {
         return import('../courses-my-lazy.module').then(m => m.CoreCoursesMyLazyModule);
     }
 
+    /**
+     * Check whether completion is available in a certain course.
+     * This is a temporary function to be used until we move AddonCourseCompletion to core folder.
```
Comment (dpalou) [inline src/core/features/courses/services/courses-helper.ts:444]: Did you open an issue in the tracker so we don't forget about this?

File: src/addons/blog/blog.module.ts [hunk 2]
```diff
@@ -22,16 +22,24 @@ import { CoreMainMenuTabRoutingModule } from '@features/mainmenu/mainmenu-tab-ro
...
+ *
+ * @returns Blog services.
+ */
+export async function getBlogServices(): Promise<Type<unknown>[]> {
+    const { AddonBlogProvider } = await import('@addons/blog/services/blog');
+
+    return [
+        AddonBlogProvider,
+    ];
+}
```
Comment (dpalou) [inline src/addons/blog/blog.module.ts:42]: Since this code isn't used in compile, maybe we should just remove it like we did for data privacy?

**Review by dpalou — CHANGES_REQUESTED** [review]

File: src/core/features/courses/services/courses-helper.ts (hunk 1 above)
Comment (crazyserver) [reply src/core/features/courses/services/courses-helper.ts:444]: Done https://tracker.moodle.org/browse/MOBILE-4537

File: src/core/features/tag/services/tag.ts [hunk 3]
```diff
@@ -53,7 +53,8 @@ export class CoreTagProvider {
     areTagsAvailableInSite(site?: CoreSite): boolean {
         site = site || CoreSites.getCurrentSite();
 
-        return !!site && site.wsAvailable('core_tag_get_tagindex_per_area') &&
+        return !!site && site.canUseAdvancedFeature('usetags') &&
```
[the remark this answers is not in the document]
Comment (crazyserver) [reply src/core/features/tag/services/tag.ts:56]: Yep, good catch!

**Review by dpalou — APPROVED** [review]

**Pull request merged**